assemble/Multiphase.F90

```diff
@@ -397,8 +397,8 @@ subroutine assemble_fluid_particle_drag(istate_fluid, istate_particle)
                   if(have_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_diameter")) then
                      call get_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_diameter", d)
                      have_constant_d = .true.
-                  else if(have_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_diameter_use_scalar_field")) then
-                     call get_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_diameter_use_scalar_field", d_field_name)
+                  else if(have_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_dia_use_scalar_field")) then
+                     call get_option(trim(state(istate_particle)%option_path)//"/multiphase_properties/particle_dia_use_scalar_field", d_field_name)
```

Comment:
Why was this option renamed? `particle_dia` sounds a bit random to me, and in general we don't use abbreviations in options names